src/EditorFeatures/CSharpTest/AutomaticCompletion/AutomaticLineEnderTests.cs

```diff
@@ -703,6 +702,144 @@ public void NotAfterAutoPropertyDeclaration()
 }");
         }
 
+        [WorkItem(150480, "devdiv.visualstudio.com")]
```

Comment:
"devdiv.visualstudio.com" doesn't appear anywhere in the codebase today. Is this how we want to mark internal bugs? (/cc: @Pilchie)